package.json

```diff
@@ -83,13 +83,15 @@
 		"missing-native-js-functions": "^1.2.18",
 		"morgan": "^1.10.0",
 		"multer": "^1.4.5-lts.1",
+		"mysql2": "^2.3.3",
 		"node-2fa": "^2.0.3",
 		"node-fetch": "^2.6.7",
 		"patch-package": "^6.4.7",
 		"picocolors": "^1.0.0",
 		"prettier": "^2.7.1",
 		"proxy-agent": "^5.0.0",
 		"reflect-metadata": "^0.1.13",
+		"sqlite3": "^5.1.1",
```

Comment:
I thought these were gonna be optional deps?
Not a deal breaker, just want context

Reply:
accident